src/mat/4C_mat_fourier.hpp

```diff
@@ -19,74 +16,70 @@
 
 FOUR_C_NAMESPACE_OPEN
 
-/*----------------------------------------------------------------------*
- |                                                           dano 09/09 |
- *----------------------------------------------------------------------*/
+/* Fourier material according to [1]
+ *
+ * This is a Fourier's law of instationary heat conduction.
+ * The anisotropic version is the extension of the isotropic case with the
+ * thermal conductivity treated as tensor.
+ *
+ * <h3>References</h3>
+ * <ul>
+ * <li> [1] GA Holzapfel, "Nonlinear solid mechanics", Wiley, 2000.
+ * </ul>
+ */
+
 namespace Mat
 {
   namespace PAR
   {
-    /*----------------------------------------------------------------------*/
-    /// material parameters for FourierIso material
+    /// material parameters for Fourier material
     ///
     /// <h3>Input line</h3>
-    /// MAT 1 THERM_FourierIsoIso CAPA 1.0 COND 1.0
-    class FourierIso : public Core::Mat::PAR::Parameter
+    /// MAT 1 MAT_Fourier CAPA 1.0 NUMSPACEDIM 3 CONDUCT 1.0 0.1 10.0
```

Comment:
I would remove this. This gets outdated and is only correct for a dat input file.